src/java.base/share/classes/sun/security/jca/Providers.java

```diff
@@ -100,6 +100,20 @@ private Providers() {
         "SunJCE",
     };
 
+    // Hardcoded fully-qualified names of providers to use for JAR
```

Comment:
These are "class" names: The comment should say so.

Reply:
Updated to indicate that.